src/openrct2-ui/windows/Error.cpp

```diff
@@ -108,9 +109,19 @@ static Widget window_error_widgets[] = {
                 Close();
             }
         }
+
+        void OnUpdate() override
+        {
+            // Automatically close previous screenshot messages before new screenshot is taken
+            if (number == 1 && gScreenshotCountdown > 0)
```

Comment:
If I understand correctly, you make the `number` member do double duty to track whether to autoclose? I think a constant would be a good idea, then.